src/org/sparkboard/server/handle.clj

```diff
@@ -1,35 +1,36 @@
 (ns org.sparkboard.server.handle
   "HTTP request handlers"
   (:require [clojure.string :as string]
-            [jsonista.core :as json]
             [org.sparkboard.server.slack.core :as slack]
             [org.sparkboard.server.slack.hiccup :as hiccup]
             [org.sparkboard.server.slack.screens :as screens]
+            [org.sparkboard.slack.slack-db :as slack-db]
+            [org.sparkboard.js-convert :refer [json->clj]]
+            [server.common :as common]
             [ring.util.http-response :as http]
             [taoensso.timbre :as log]))
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;;; Utility fns
 
-(defn read-json [v]
-  (json/read-value v (json/object-mapper {:decode-key-fn keyword})))
-
 (defn decode-text-input [s]
   ;; Slack appears to use some (?) of
   ;; `application/x-www-form-urlencoded` for at least multiline text
   ;; input, specifically replacing spaces with `+`
   (string/replace s "+" " "))
 
-(defn request-updates [msg reply-channel]
+(defn request-updates [{:keys [slack/token]} msg reply-channel]
   ;; TODO Write broadcast to Firebase
   (log/info "[request-updates] msg:" msg)
-  (mapv #(slack/web-api "chat.postMessage"
-                         {:channel %
-                          :blocks (hiccup/->blocks-json (screens/team-broadcast-message msg reply-channel))})
-        (keep (fn [{:strs [is_member id]}]
-                ;; TODO ensure bot joins team-channels when they are created
-                (when is_member id))
-              (get (slack/web-api "channels.list") "channels"))))
+  (let [blocks (hiccup/->blocks-json (screens/team-broadcast-message msg reply-channel))]
+    (mapv #(slack/web-api "chat.postMessage"
+                          {:channel %
+                           :blocks blocks
+                           :slack/token token})
+          (keep (fn [{:strs [is_member id]}]
+                  ;; TODO ensure bot joins team-channels when they are created
+                  (when is_member id))
+                (get (slack/web-api "channels.list" {:slack/token token}) "channels")))))
```

Comment:
Since the token isn't going in the body, in my follow-up PR I may move it to the first position, separate from the body, as a new parameter.